db/seeds.rb

```diff
@@ -153,6 +158,8 @@
     assign: "Matheus Roberto",
     pipeline: "In Progress",
     initial_date: "01/01/2017",
+    final_date: "04/01/2017",
```

Comment:
As histórias que tem o pipeline "In Progress" não fazem sentido ter final_date

Reply:
Feito!